tasks/vendoring/__init__.py

```diff
@@ -164,7 +164,8 @@ def download_licenses(ctx, vendor_dir):
     log('Downloading licenses')
     tmp_dir = vendor_dir / '__tmp__'
     ctx.run(
-        'pip download -r {0}/vendor.txt --no-binary :all: -d {1}'.format(
+        'pip download -r {0}/vendor.txt --no-binary '
+        '--no-deps :all: -d {1}'.format(
```

Comment:
err, I was going to merge this but this ended in the wrong commit (I guess you must have amended this change into the last commit :)
And more importantly, `--no-binary --no-deps :all`  won't work, you need `--no-binary :all --no-deps`

Reply:
changed.